eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -0,0 +1,241 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules
```

Comment:
out of curiosity does powershell export the class PackageProps? What does it look like on the consumption side?

Reply:
It does not export the class. I think I will have to explicitly indicate that the module should export the class